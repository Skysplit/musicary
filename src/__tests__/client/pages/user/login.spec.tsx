import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '@client/pages/user/login';

describe('<LoginPage />', () => {
  test('should render page', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
