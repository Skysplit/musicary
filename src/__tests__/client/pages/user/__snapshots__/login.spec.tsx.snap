// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> should render page 1`] = `
<Card
  animate={true}
  expanderIcon={
    <FontIcon
      iconClassName="material-icons"
    >
      keyboard_arrow_down
    </FontIcon>
  }
  expanderTooltipPosition="left"
>
  <CardTitle
    title="Login"
  />
  <CardText
    component="section"
  >
    <Connect(LoginFormContainer) />
  </CardText>
</Card>
`;
